webext/raptor/measure.js

```diff
@@ -11,33 +11,53 @@ var gRetryCounter = 0;
 var getHero = false;
 var heroesToCapture = [];
 
-// measure time-to-first-non-blank-paint
+// measure firefox time-to-first-non-blank-paint
 // note: this browser pref must be enabled:
 // dom.performance.time_to_non_blank_paint.enabled = True
 // default only; this is set via control server settings json
 var getFNBPaint = false;
 
+// measure google's first-contentful-paint
+// default only; this is set via control server settings json
+var getFCP = false;
```

Comment:
I assume `FNBPaint` and `FCP` are measuring the same thing just on different browsers? It would be good to abstract these away behind a single `first-paint`  event.

I'm probably using the wrong nomenclature here, so feel free to tweak it. But you could create an 'events' folder and each thing you want to measure could have its own file. E.g, `events/first-paint.js`. Then in that file you would have all the different implementation details for the various browsers under test (it's prudent to assume that there will be more browsers added in the future). That way the controller (measure.js) would be completely browser independent and we could extend it, either with more events or more browsers.

Feel free to file a follow-up for this though.